lib/tests/live.rs

```diff
@@ -6,8 +6,8 @@ use surrealdb::dbs::Session;
 use surrealdb::err::Error;
 use surrealdb::sql::Value;
 use surrealdb_core::fflags::FFLAGS;
-use surrealdb_core::kvs::LockType::Optimistic;
-use surrealdb_core::kvs::TransactionType::Write;
+
+// RUST_LOG=trace cargo test -p surrealdb --features kv-mem --test live -- --nocapture
```

Comment:
Do we need to keep this?

Reply:
Will follow up so as not to lose ticks

Reply:
This is being removed in the february lint fixup